README.md

```diff
@@ -164,6 +166,95 @@ geographic region in support of yourproject’s security and compliance needs.
 See [Running in particular zone/region doc](docs/running_in_particular_zone_region.md).
 
 
+## Running VCF files to BigQuery Preprocessor
+
+Similarly, the preprocessor can be ran using docker or directly from the source.
```

Comment:
Please add a sentence or two about what this tool does and why someone should use it.